packages/fx-core/src/plugins/solution/fx-solution/debug/util/tasksNext.ts

```diff
@@ -51,7 +52,13 @@ export function generateTasks(
   }
 
   if (includeBot) {
-    tasks.push(startBot(includeFrontend));
+    if (includeFuncHostedBot) {
+      tasks.push(startFuncHostedBot(includeFrontend, programmingLanguage));
+      tasks.push(startAzuriteEmulator());
+      tasks.push(watchFuncHostedBot());
```

Comment:
No need to add watch for ts?

Reply:
I believe ts requires watch, but js does not.